### ISSUE TITLE
Hey Jack, it's time to transfer :)

### ISSUE THREAD

**Issue by levithomason** [Semantic-Org/Semantic-UI-React#3]
@jlukic We're finally ready to move Stardust here.  So pumped about how it turned out and where it's going from here.  You can checkout the [repo](https://github.com/TechnologyAdvice/stardust) and [docs](technologyadvice.github.io/stardust/) to get the gist.

### What I need

I want to delete this repo and transfer the current repo here under the same name.  In order for this to work, the Owner (you) will have to accept the transfer email within 1 day of my initiating the transfer.  I want to coordinate this so we don't miss the window and to minimize any down time for links to either repo.

Please respond here and I'll start the transfer, you can then delete this repo and accept the new one.  I'll be a collaborator after that, so I'll need bumped back up to an admin.

Cheers!

**Comment by jcarbo** [comment]
Any updates here? Eager to officially get this project into the SUI family 😄

**Comment by levithomason** [comment]
Definitely, @jlukic bump.  I'll see about emailing him directly as well.

**Comment by levithomason** [comment]
Direct email sent, we'll see.  I've also requested more access to prevent this kind of outage / blocker in the future.

**Comment by jlukic** [comment]
I can accept the transfer if you send over today or tomorrow. Just give me a headsup when its sent

**Comment by levithomason** [comment]
Cool, I'll initiate it right now with the name `stardust`.  Then we can delete this repo.  Finally, we'll do all the rename rewiring to make `starudst` => `Semantic-UI-React`.

**Comment by levithomason** [comment]
Correction, apparently I can't even start a request without being a Semantic-Org admin:

![image](https://cloud.githubusercontent.com/assets/5067638/19131996/89122174-8b06-11e6-9ecc-7e18cb4bd885.png)

Little help @jlukic?

**Comment by jlukic** [comment]
I'm not comfortable giving permanent full ownership  to Semantic Org.  I am willing however to temporarily give owners permissions for the duration of the transfer, with the understanding that afterwards you'd remove yourself from the owner's group.

With the react team admin rights you'll continue to have ability to add collaborators and administer the react repo.

**Comment by levithomason** [comment]
I get that.  Let's go ahead with your proposal for temporary admin access.  I'll remove myself after it is done from the main org but maintain access to the React repo/team.

**Comment by jlukic** [comment]
Ok you have full owner permission, should be all set.

**Comment by levithomason** [comment]
Cool, mid way through.  Will report when done.
